### DISCUSSION TITLE
Passing javascript functions to Quasar components?

### DISCUSSION THREAD

**Discussion by aalexei** [elimintz/justpy#339] [Q&A]
Is there a way to pass a javascript function to a Quasar component?

e.g. a QTable column can take a format attribute as a function like `(val, row) => '${val}%'` however setting a suitable string doesn't seem to cut it with the console reporting an error `TypeError: t.format is not a function`

**Comment by elimintz** [comment]
Hi,

I haven't tested this but try the following:
```python
t = jp.QTable()
t.evaluate_prop = ['format']
```

This tells the frame work to evaluate the prop in the front end when assigning it. For example, it works with the `rules` prop of QInput

Please let me know if this works.
